src/mac.m

```diff
@@ -197,6 +197,11 @@
     return PUGL_SUCCESS;
   }
 
+  if (drawViewResize) {
```

Comment:
I don't think this is an appropriate place to be messing with the views, before this function did just what it said on the tin, now it's `dispatchCurrentChildViewConfigurationExceptSometimesMaybeResizeTheDrawView`.

Reply:
Come to think of it, at a higher level, doing this in `dispatchExpose` seems like a kludge.  This should have happened earlier, although I'm not sure where off the top of my head.